lib/components/viewers/route-viewer.js

```diff
@@ -83,14 +115,64 @@ class RouteViewer extends Component {
       agencies,
       filter,
       findRoute,
+      getVehiclePositionsForRoute,
       hideBackButton,
       intl,
       modes,
       routes: sortedRoutes,
       setViewedRoute,
       transitOperators,
-      viewedRoute
+      viewedRoute,
+      viewedRouteObject
     } = this.props
+
+    const { initialRender } = this.state
+    const { ModeIcon } = this.context
+
+    // If patternId is present, we're looking at a specific pattern's stops
+    if (viewedRoute?.patternId && viewedRouteObject) {
+      const { patternId } = viewedRoute
+      const route = viewedRouteObject
+      // Find operator based on agency_id (extracted from OTP route ID).
+      const operator =
+        coreUtils.route.getTransitOperatorFromOtpRoute(
+          route,
+          transitOperators
+        ) || {}
+
+      return (
+        <div className='route-viewer'>
+          {/* Header Block */}
+          <div
+            className='route-viewer-header'
+          >
+            {/* Always show back button, as we don't write a route anymore */}
```

Comment:
Don't need to spread the opening `<div>` over multiple lines.

Reply:
Good call. Prettier is doing some weird things in this repo. Fixed in 9a7139548d78db23db07dfddf9940bd85078072b